index.js

```diff
@@ -223,6 +223,10 @@ function buildFetch(behavior) {
             return handleError(url, cacheKey, err.response, err.response.body, resolvedCallback);
           }
         } else if (err instanceof got.TimeoutError) {
+          if (Object.keys(options.timeout).includes("request") && Object.keys(options.timeout).includes("socket")) {
+            const { timeout: { socket, request: req } } = options;
+            logger.info(`Expected server response is greater than timeout option for socket: ${socket} ms and request: ${req} ms`);
```

Comment:
This not info, this is debug

Reply:
and the information is pretty useless. Wouldn't it be better to output the timings object along the configured timeout? Otherwise you can't determine _why_ it timed out anyways.

Reply:
Yes I agree. Please check the new code.